GLTFSDK/GLTFSDK.CPP.autopkg

```diff
@@ -1,7 +1,18 @@
+configurations {
+    Platform {
+        key : "Platform";
+        choices: { Win32, x64, ARM, ARM64, AnyCPU };
+        Win32.aliases : { x86, win32, ia32, 386 };
+        x64.aliases : { x64, amd64, em64t, intel64, x86-64, x86_64 };
+        ARM.aliases : { arm, woa };
+        ARM64.aliases : { arm64 };
+        AnyCPU.aliases : { anycpu, any };
+    };
+}
 nuget {
     nuspec {
         id = Microsoft.glTF.CPP;
-        version : 1.2.16.0;
+        version : 1.2.17.0;
```

Comment:
This isn't used anymore. We should probably zero it out.

Reply:
Good call, done!